roles/advanced-core/advanced_dns_server/vars/CentOS_8.yml

```diff
@@ -0,0 +1,8 @@
+---
+advanced_dns_server_packages_to_install:
+  - bind
+  - python3-netaddr
```

Comment:
Why do we need this rpm on the dns server?

Reply:
Because it is used in its templates:

    {% set addresses = networks[network]['services_ip']['dns_ip'] | ipaddr %}

To make the role independent, I need to add it.

Reply:
This is needed on the ansible controller, not on the dns server.

Reply:
It may happen both ansible controller and dns server are on the same host, but this package is not needed in the dns role, it is needed in the bluebanquise role (to configure the ansible controller for bluebanquise).

Reply:
In fact you are right. I will remove it.